homeassistant/components/here_travel_time/__init__.py

```diff
@@ -34,12 +34,12 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     arrival = (
         dt.parse_time(config_entry.options[CONF_ARRIVAL_TIME])
-        if config_entry.options[CONF_ARRIVAL_TIME] is not None
+        if config_entry.options.get(CONF_ARRIVAL_TIME) is not None
```

Comment:
You could do `dt.parse_time(config_entry.options.get(CONF_ARRIVAL_TIME, ""))` instead as that would return None